src/libexpr/primops.cc

```diff
@@ -846,11 +846,32 @@ static RegisterPrimOp primop_floor({
     .fun = prim_floor,
 });
 
+class Counter
```

Comment:
There already is a `MaintainCount` class in util.hh that does the same thing.

Reply:
switched to MaintainCount